### ISSUE TITLE
Documentation for GUI

### ISSUE THREAD

**Issue by Zumbalamambo** [intel-isl/Open3D#2543]
Please add the documentation for `GUI`. I have tried to follow along with `vis_gui.py` but it's hard to follow it. I would like to find a way to use `set_look_at`, `set_front`, `set_zoom` options but I'm not able to find a way to do that :(

**Comment by 1939938853** [comment]
Yes. 

It is very good to provide some documentation and tutorial for both rendering and gui. For me, I'm especially looking forward to the tutorials for rendering and gui in C++. 

Now there are lots of new things in rendering, only studying the API and source code, like the one open3d\visualization\visualizer\GuiVisualizer.cpp, is a little bit difficult. 

Is it possible to realize PBR rendering without imgui? 

Thank you!

**Comment by germanros1987** [comment]
We are working on it.

**Comment by hduonggithub** [comment]
@germanros1987 
Do you know how to get the ViewControl in this new GUI to set the following?
set_look_at, 
set_front, 
set_zoom

With non-gui visualizer, I can get it by vis.get_view_control(). I am looking for a method in the new GUI. Thanks!

**Comment by ssheorey** [comment]
Documentation for the new filament based visualization: [`open3d.visualization.gui`](http://www.open3d.org/docs/latest/python_api/open3d.visualization.gui.html) and [`open3d.visualization.rendering`](http://www.open3d.org/docs/latest/python_api/open3d.visualization.rendering.html) is available. We are still working on adding more functionality.

This may be useful: [`open3d.visualization.rendering.Camera.look_at()`](http://www.open3d.org/docs/latest/python_api/open3d.visualization.rendering.html)

**Comment by hduonggithub** [comment]
@ssheorey 

I tried to set 3 values for look_at(), but I got this incompatible error? I am using open3d  0.13.0. Any idea? Thanks

rendering.Camera.look_at((-2.25072473, -0.40637109,  1.09599469), (0, 0, 1), (0, 1, 0))
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: look_at(): incompatible function arguments. The following argument types are supported:
    1. (self: open3d.cpu.pybind.visualization.rendering.Camera, arg0: numpy.ndarray[float32[3, 1]], arg1: numpy.ndarray[float32[3, 1]], arg2: numpy.ndarray[float32[3, 1]]) -> None

Invoked with: (-2.25072473, -0.40637109, 1.09599469), (0, 0, 1), (0, 1, 0)

**Comment by ssheorey** [comment]
You need to setup the camera for the specific 3D scene that you have created. See here for examples:

https://github.com/intel-isl/Open3D/blob/2674180540c1fb9437e726c487e69097371e53d9/examples/python/gui/online-processing.py#L420
